### ISSUE TITLE
Feature Request getString()

### ISSUE THREAD

**Issue by sstaub** [CNMAT/OSC#155]
A `const char* getString(int position)` function will helpful, this allows to pass without copying the buffer data.

**Comment by adrianfreed** [comment]
I will look at a PR. I assume you don't want to know the length of the string and can handle its temporary status?

**Comment by sstaub** [comment]
Yes, I know about the problem. I use buffers with a fixed length when I know the maximum length of the incoming data.
